data/core/src/commonMain/kotlin/com/tunjid/heron/data/utilities/multipleEntitysaver/SaveLabeler.kt

```diff
@@ -0,0 +1,29 @@
+package com.tunjid.heron.data.utilities.multipleEntitysaver
+
+import app.bsky.labeler.LabelerViewDetailed
+import com.tunjid.heron.data.core.types.GenericUri
+import com.tunjid.heron.data.core.types.ProfileId
+import com.tunjid.heron.data.database.entities.LabelerEntity
+
+internal fun MultipleEntitySaver.add(
+    labeler: LabelerViewDetailed,
+) {
+    val creator = labeler.creator
```

Comment:
You should also be calling `MultipleEntityProvider.add(viewingProfileId, creator)` here from [`SaveProfileView.kt`](https://github.com/tunjid/heron/blob/93d5ad796cc35208516697e18c58415aaacede01/data/core/src/commonMain/kotlin/com/tunjid/heron/data/utilities/multipleEntitysaver/SaveProfileView.kt#L4).